src/components/Tour/TourButton.js

```diff
@@ -8,7 +8,10 @@ export const TourButton = () => {
   const dispatch = useDispatch();
 
   return (
-    <button onClick={() => dispatch(tourShown())} className="tour-button">
+    <button
+      onClick={() => dispatch(tourShown())}
+      className="tour-button a-btn"
```

Comment:
Trivial, but I would swap these so it's `a-btn tour-button` so it goes general -> specific. `a-btn` is a widely used class across the codebase and `tour-button` is specific to this use-case. If we ever made `tour-button` a modifier on `a-btn`, it would be `a-btn a-btn__tour` or whatever, so it would already be set up for that.